templates/annotate.sh

```diff
@@ -124,6 +127,72 @@ vep () {
   !{singularity_vep} vep "${args[@]}"
 }
 
+capice () {
+  local vcfPath="!{vcfPath}"
```

Comment:
what is the use of this line? you already have the template variable vcfPath available

Reply:
For the "`${...%%.*}` magic". It doesn't work if I write it like this: `"${\"!{vcfPath}\"%%.*}_prepared.vcf.gz"`